packages/react-preferences/src/components/BusinessFooter.tsx

```diff
@@ -35,8 +35,13 @@ export const BusinessFooter: React.FunctionComponent<{
   links: any;
   theme: string;
 }> = ({ links, theme }) => {
-  if (Object.keys(links).length === 0) {
-    return null;
+  if (
+    typeof links !== "object" ||
```

Comment:
This looks good, but the specific case we're trying to prevent is if a link exists but is has an empty string.

Once theres handling around that we should be good to go

`
{
   "links":{
      "facebook":{
         "url":""
      }
   }
}
`